MvvmCross/Platforms/Ios/Views/MvxBaseViewController.cs

```diff
@@ -185,6 +185,7 @@ protected virtual void OnKeyboardChanged(bool visible, CGRect keyboardFrame)
             }
             else
             {
+                keyboardFrame.Height -= scrollView.SafeAreaInsets.Bottom;
```

Comment:
Please add a check to see if SafeAreaInsets is available. It isn't on all iOS versions.

Reply:
Will do. I'm talking to my employer about the license agreement too.